blog_api/lib/blog_api_web/controllers/users_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule BlogApiWeb.UsersController do
+  use BlogApiWeb, :controller
+
+  alias BlogApiWeb.Auth.Guardian
+
+  action_fallback(BlogApiWeb.FallbackController)
```

Comment:
É só uma preferência minha mesmo 😅 
```suggestion
  action_fallback BlogApiWeb.FallbackController
```